client/ayon_houdini/plugins/publish/collect_slapcomps.py

```diff
@@ -0,0 +1,92 @@
+import hou
+import pyblish.api
+
+from ayon_houdini.api import plugin
+
+
+class CollectSlapComps(plugin.HoudiniInstancePlugin):
+    """Collect Slap Comps.
+
+    It collects Slap Comps from the USD render rop node and
+    save them as a list in `slapComp` key in `instance.data`.
+
+    Each item in the list follows:
+        slap_comp_path?option=value&option2=value2
+
+    """
+
+    order = pyblish.api.CollectorOrder
+    label = "Collect Slap Comps"
+    families = ["usdrender"]
+
+    def process(self, instance):
+        if not instance.data["farm"]:
+            return
+
+        node_path = instance.data.get("instance_node")
+        if node_path is None:
+            # Instance without instance node like a workfile instance
+            self.log.debug(
+                "No instance node found for instance: {}".format(instance)
+            )
+            return
+
+        ropnode = hou.node(node_path)
+
+        slapcomp_sources =  []
+
+        comp_numbers = ropnode.evalParm("husk_slapcomp")
+        for i in range(1, comp_numbers+1):
+        
+            if ropnode.evalParm(f'husk_sc_enable{i}'):
+
```

Comment:
This could be less indented overall if you make this:


```suggestion
            if not ropnode.evalParm(f'husk_sc_enable{i}'):
                continue
```

And then unindent the below.